### PR TITLE
My page login

### PR DESCRIPTION
**Pull request by andrnors**
One main users have multiple subusers, and can switch in between them. Everything seems to work.

### CODE REVIEW

File: skalvi/models.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class UserProfile(models.Model):
     type = models.CharField(max_length=5)  # A = admin, P = parent, C = child, etc.
     phone = models.CharField(max_length=8, null=True, blank=True)  # 90 90 99 09 <-- gives length 8
     profile_name = models.CharField(max_length=30)
+    is_active = models.BooleanField(default=False)
```
Comment (skaugvoll) [inline skalvi/models.py:18]: Why is default = False? Is that smart? won't the user be banned though?

File: skalvi/templates/chooseUser.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<style>
```
Comment (skaugvoll) [inline skalvi/templates/chooseUser.html]: extract to own stylesheet

File: skalvi/models.py (hunk 1 above)
Comment (andrnors) [reply skalvi/models.py:18]: No, it is a separate field. It is to show which user are logged in.

File: skalvi/templates/chooseUser.html (hunk 2 above)
Comment (andrnors) [reply skalvi/templates/chooseUser.html]: Will do

File: skalvi/models.py (hunk 1 above)
Comment (skaugvoll) [reply skalvi/models.py:18]: Can only one of the "user group" be signed in at a given time, our can multiple, say father and child be active simultaneously?

Comment (andrnors) [reply skalvi/models.py:18]: At this time, only one

Comment (skaugvoll) [reply skalvi/models.py:18]: Create a issue on multiple users simultaneously :)

**Pull request merged**